versioned_docs/version-7.x/stack-navigator.md

```diff
@@ -433,6 +478,18 @@ Accessibility label for the header back button.
 
 Function which returns a React Element to display custom image in header's back button. When a function is used, it receives the `tintColor` in it's argument object. Defaults to Image component with back image source, which is the default back icon image for the platform (a chevron on iOS and an arrow on Android).
 
+<img src="/assets/7.x/stack/headerBackImage.png" width="500" alt="Header back image" />
+
+```js
+      headerBackImage: ({ tintColor }) => (
+       <Ionicons
+          tyle={{ color: tintColor }}
+          size={26}
+          name="arrow-back-circle"
+        />
+      ),
+```
```

Comment:
Same as above about correcting the example to contain Screen definition.

```suggestion
Example:
```js
      headerBackImage: ({ tintColor }) => (
       <Ionicons
          tyle={{ color: tintColor }}
          size={26}
          name="arrow-back-circle"
        />
      ),
```
```